src/Handler/FatalErrorHandler.php

```diff
@@ -14,18 +14,16 @@
 
 use ErrorException;
 use Throwable;
-use WeCodeIn\ErrorHandling\Processor\ProcessorInterface;
 
 class FatalErrorHandler extends AbstractErrorHandler
 {
-    use ProcessableTrait;
-
     private $allocatedMemory;
 
-    public function __construct(int $allocatedMemorySize, ProcessorInterface ...$processors)
+    public function __construct(int $allocatedMemorySize, array $processors = [])
```

Comment:
I prefer ProcessorInterface ...$processors instead of array $processors = [] as you can later when you creating instance of AbstractProcessingHandler implementation pass any iterable easy

eg.
```
$queue = new SplQueue();
$queue->enqueue($processor1);
$queue->enqueue($processor2);

$handler = new FatalErrorHandler(10, ...$queue);
```